### PR TITLE
FIX FOR JS ISSUES

### PR DESCRIPTION
**Pull request by biyeun**
##### SUMMARY
It looks like appcues might have changed how their module behaves around AMD setups since when we first implemented it and now, and this has been causing intermittent failures on staging and prod.

Sometimes (or often???) the appcues script loads before the final js bundle (being triggered by `analytix/appcues` in the definition of `base_main`), causing a `requirejs` mismatch error to throw, subsequently causing the final bundle modules to not execute properly even though the bundle itself loads. This is because appcues now detects if an AMD system like `requirejs` is present and defines itself as a module once the script gets loaded onto the page. This, of course, messes up how we then deal with the `Appcues` global, and might even cause `analytix/appcues` to try and load the `appcues` js script again because it can't find the global.

Anyway, long story short, following the [first set of instructions here](https://docs.appcues.com/article/303-using-appcues-with-requirejs) fixes the issue. It skips appcues loading as a module (since some pages using it are not on require.js yet).

The good news in all of this? At least it didn't affect ICDS...lol

### CODE REVIEW

**Comment by biyeun** [comment]
killed the build on purpose

**Comment by dannyroberts** [comment]
🔥

**Review by dannyroberts — APPROVED** [review]

**Comment by orangejenny** [comment]
🍰

**Review by millerdev — COMMENTED** [review]
🏅 Excellent!
